example/src/main/java/com/xwray/groupie/example/MainActivity.kt

```diff
@@ -121,6 +121,27 @@ class MainActivity : AppCompatActivity() {
             }
         }
 
+        // Reordering a Section of Expandable Groups
+        val section = Section(HeaderItem(R.string.reorderable_section))
+        val swappableExpandableGroup = mutableListOf<ExpandableGroup>()
+        for (i in 0..1) {
+            val header = ExpandableHeaderItem(R.string.reorderable_item_title, R.string.reorderable_item_subtitle)
+            val group = ExpandableGroup(header).apply {
+                for (j in 0..1) {
```

Comment:
to make the reordering clearer, can you make one of the groups have more items than the other?

Reply:
I tried to make it obvious with colors - but I like this suggestion 👍